<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MindMapClass</class>
 <widget class="QMainWindow" name="MindMapClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MindMap</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
    <addaction name="actionInsert_a_Child"/>
    <addaction name="actionInsert_a_Sibling"/>
    <addaction name="actionInsert_a_Parent"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionInsert_a_Child"/>
   <addaction name="actionInsert_a_Sibling"/>
   <addaction name="actionInsert_a_Parent"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEdit">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icon/pencil41.png</normaloff>Resources/icon/pencil41.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icon/delete96.png</normaloff>Resources/icon/delete96.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionInsert_a_Child">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icon/round77.png</normaloff>Resources/icon/round77.png</iconset>
   </property>
   <property name="text">
    <string>Insert a Child</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionInsert_a_Parent">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icon/rounded54.png</normaloff>Resources/icon/rounded54.png</iconset>
   </property>
   <property name="text">
    <string>Insert a Parent</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionInsert_a_Sibling">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icon/square181.png</normaloff>Resources/icon/square181.png</iconset>
   </property>
   <property name="text">
    <string>Insert a Sibling</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icon/add26.png</normaloff>Resources/icon/add26.png</iconset>
   </property>
   <property name="text">
    <string>Create a new mind map</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icon/folder232.png</normaloff>Resources/icon/folder232.png</iconset>
   </property>
   <property name="text">
    <string>Open a mind map</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icon/floppy1.png</normaloff>Resources/icon/floppy1.png</iconset>
   </property>
   <property name="text">
    <string>Save a mind map</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icon/scissors9.png</normaloff>Resources/icon/scissors9.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icon/copy8.png</normaloff>Resources/icon/copy8.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icon/clipboard46.png</normaloff>Resources/icon/clipboard46.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GraphicalUI.qrc"/>
 </resources>
 <connections/>
</ui>
